flask/app/tissue_samples.py

```diff
@@ -54,7 +62,16 @@ def get_tissue_sample(id: int):
     return jsonify(
         {
             **asdict(tissue_sample),
-            "datasets": tissue_sample.datasets,
+            "created_by": tissue_sample.created_by.username,
+            "updated_by": tissue_sample.updated_by.username,
+            "datasets": [
+                {
+                    **asdict(dataset),
+                    "created_by": dataset.created_by.username,
+                    "updated_by": dataset.updated_by.username,
+                }
+                for dataset in tissue_sample.datasets
+            ],
```

Comment:
Yeah, this segment here does perform two additional queries per dataset, so can we leave it as is as `"datasets": tissue_sample.datasets`